src/index.ts

```diff
@@ -10,7 +10,7 @@ export interface HandlebarsPluginConfig {
   context?: Record<string, unknown>;
   compileOptions?: CompileOptions;
   runtimeOptions?: RuntimeOptions;
-  partialDirectory?: string;
+  partialDirectory?: Array<string>;
```

Comment:
So this would end up looking something like this instead

```suggestion
  partialDirectory?: string | Array<string>;
```